scripts/uninstall.sh

```diff
@@ -2,11 +2,9 @@
 set -x
 
 ROOT_DIR=$(dirname $0)/..
-NAMESPACE=${NAMESPACE:-examples}
+NAMESPACE=${NAMESPACE:-object-detection}
```

Comment:
Revert this change.

Reply:
made a separate folder for original scripts